app/experimenter/nimbus-ui/src/components/PageNew/index.test.tsx

```diff
@@ -40,33 +96,67 @@ describe("PageNew", () => {
     expect(global.console.log).toHaveBeenCalledWith("CANCEL TBD");
   });
 
-  const Subject = () => {
-    return <PageNew />;
+  const mkGqlMocks = ({
+    message = "success" as string | Record<string, any>,
+    status = 200,
+    nimbusExperiment = { ...mockSubmit, slug: "foo-bar-baz" },
+  } = {}) => [
+    {
+      request: {
+        query: CREATE_EXPERIMENT_MUTATION,
+        variables: {
+          input: mockSubmit,
+        },
+      },
+      result: {
+        errors: undefined as undefined | any[],
+        data: {
+          createExperiment: {
+            clientMutationId: "8675309",
+            message,
+            status,
+            nimbusExperiment,
+          } as any | undefined,
```

Comment:
Once types are automatically generated it should be possible to swap out `any` I think?

Reply:
This was to accommodate an intentional breakage in a test to cause an error. Reworked it a bit to make that clearer closer to where it happens.